internal/app/machined/pkg/controllers/k8s/manifest_test.go

```diff
@@ -171,9 +178,7 @@ func (suite *ManifestSuite) TestReconcileKubeProxyExtraArgs() {
 	rootSecrets := secrets.NewRoot(secrets.RootKubernetesID)
 	manifestConfig := config.NewK8sManifests()
 	spec := defaultManifestSpec
-	spec.ProxyExtraArgs = map[string]string{
-		"bind-address": "\"::\"",
-	}
+	spec.ProxyArgs = append(spec.ProxyArgs, "--bind-address=\"::\"")
 	manifestConfig.SetManifests(spec)
```

Comment:
any high-level tests here to make sure args building process goes fine with overrides?

Reply:
I've added some